src/index.html

```diff
@@ -11,9 +11,56 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet">
 </head>
 <body>
-
-<script type="text/javascript" src="./main.js"></script>
+  <header class="navigate">
+    <img class="logo-img" src="./images/logo.png" alt="logo_moyo">
+    <nav>
+      <ul class="nav-links">
+        <li class="list">
+          <a href="#" class="is-active section">
+            apple
+          </a>
+        </li>
+        <li class="list">
+          <a href="#" class="section">
```

Comment:
section is too general name, choose a name that describe this element's behavior

Reply:
renamed.
Done.